core/domain/story_domain.py

```diff
@@ -182,6 +211,70 @@ def create_default_story_node(cls, node_id):
         """
         return cls(node_id, [], [], [], '', None)
 
+    def validate(self):
+        """Validates various properties of the story node.
+
+        Raises:
+            ValidationError: One or more attributes of the story node are
+            invalid.
+        """
+        if self.exploration_id:
+            if not isinstance(self.exploration_id, basestring):
+                raise utils.ValidationError(
+                    'Expected exploration ID to be a string, received %s' %
+                    self.exploration_id)
+
+        if not isinstance(self.outline, basestring):
+            raise utils.ValidationError(
+                'Expected outline to be a string, received %s' %
+                self.outline)
+
+        if not isinstance(self.id, basestring):
+            raise utils.ValidationError(
+                'Expected node ID to be a string, received %s' %
+                self.id)
+
+        if not isinstance(self.prerequisite_skill_ids, list):
```

Comment:
Think about interactions between the attrs. For example I guess you don't want prerequisite_skill_ids and acquired_skill_ids to have a nonempty intersection, for any given node?